src/views/Transfer/Transfer.js

```diff
@@ -122,9 +131,9 @@ export default {
                 this.$flashMessage.info(
                     `Successfully transferred ${this.formObject.amount} to ${this.formObject.to.address}.`
                 );
-            } catch {
+            } catch(e) {
                 //@todo show correct error message for multiple errors eg: "reason": "invalid address"
-
+                console.log(e)
                 this.$flashMessage.error(`Payment failed.`);
```

Comment:
We should make a better error message based on the error

Reply:
should be different issue, there is a lot to handle there